### PR TITLE
docs(stroybook-refactoring): refactor `p-badge` & `p-collapsible-list`

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description
- storybook refactoring
- change `p-badge` component to single-file-component
- change `p-collapsible-list` two way binding hook

### CODE REVIEW

File: src/data-display/badges/PBadge.vue [hunk 1]
```diff
@@ -1,19 +1,26 @@
+<template>
+    <span class="p-badge" :class="allBodyClass" :style="[inlineStyles]">
+        <slot name="default" />
```
Comment (sulmoJ) [inline src/data-display/badges/PBadge.vue]: For the default slot, you do not need to create a name.

**Review by sulmoJ — COMMENTED** [review]
LGTM!
I left a simple comment. I kept watching it during development, so I think it would be good to proceed after other people's approbations!

File: src/data-display/badges/PBadge.vue (hunk 1 above)
Comment (piggggggggy) [reply src/data-display/badges/PBadge.vue]: Thank you for the review! I edited the slot name you said.

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**